V3_MIGRATION_GUIDE.md

```
@@ -1083,6 +1095,46 @@ class MockAuthentication: Authentication {
+class MockMyAccountAuthenticationMethods: MyAccountAuthenticationMethods {
+    // ... existing method implementations
+
+    func enrollPassword(userIdentityId: String?, connection: String?) -> any Requestable<PasswordEnrollmentChallenge, MyAccountError> {
+        fatalError("not implemented")
+    }
+
+    func confirmPasswordEnrollment(id: String, authSession: String, newPassword: String) -> any Requestable<AuthenticationMethod, MyAccountError> {
+        fatalError("not implemented")
+    }
+}
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)